yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -133,7 +133,7 @@ func assessMigration() (err error) {
 
 	err = runAssessment()
 	if err != nil {
-		return fmt.Errorf("failed to run assessment: %w", err)
+		log.Errorf("failed to run assessment: %v", err)
```

Comment:
Hi @shaharuk-yb, we are currently displaying `FailureReasoning` in the report. How about `Printf` the FailureReasoning on the console also.